index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mens Portfolio</title>
+    <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+
+<div class="flex_container">
+    <nav class="flex_nav">
+        <a href="" class="toggleNav"> 
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+        </a>
+        <ul>
+            <li class=""><a href="#">Hello</a></li>
+            <li class=""><a href="#">Portfolio</a></li>
+            <li class=""><a href="#">About</a></li>
+            <li class=""><a href="#">Contact</a></li> 
+        </ul>
+    </nav>
+
+    <div class="mobile_logo">
+        <img src="img/mobile_logo.png" alt="">
+    </div>

```

Comment:
Please use `background-image` for this image in the .css file. Since decorations are always background images, not `<img>` tags. You can refer more[ here](https://www.w3schools.com/cssref/pr_background-position.asp).